packages/tealdeer/build.sh

```diff
@@ -3,7 +3,7 @@ TERMUX_PKG_DESCRIPTION="A very fast implementation of tldr in Rust"
 TERMUX_PKG_LICENSE="Apache-2.0, MIT"
 TERMUX_PKG_LICENSE_FILE="LICENSE-APACHE, LICENSE-MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1.5.0
+TERMUX_PKG_VERSION=1.6.1
 TERMUX_PKG_REVISION=1
```

Comment:
Please remove the line `TERMUX_PKG_REVISION=1` so that revision is reset to 0

Reply:
remove the line completely? or just change it to 0?

Reply:
Remove completely